openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js

```diff
@@ -499,22 +573,62 @@ describe('Glossary page should work properly', () => {
     cy.get('[data-testid="tag-selector"]')
       .should('be.visible')
       .click()
-      .type(`${glossary}.${term}`);
-    cy.get(`[title*="${term}"]`).should('be.visible').click();
+      .type(`${glossary}.${term1}`);
+    cy.get(`[title*="${term1}"]`).should('be.visible').click();
     cy.get(
       '[data-testid="tags-wrapper"] [data-testid="tag-container"]'
-    ).contains(term);
+    ).contains(term1);
 
-    interceptURL('GET', '/api/v1/feed/count*', 'saveTag');
+    cy.get('[data-testid="tag-selector"]')
+      .should('be.visible')
+      .click()
+      .type(`${glossary}.${term2}`);
+    cy.get(`[title*="${term2}"]`).should('be.visible').click();
+    cy.get(
+      '[data-testid="tags-wrapper"] [data-testid="tag-container"]'
+    ).contains(term2);
+
+    interceptURL('GET', '/api/v1/feed/count*', 'countTag');
     interceptURL('GET', '/api/v1/tags', 'tags');
+    interceptURL('PATCH', '/api/v1/tables/*', 'saveTag');
 
     cy.get('[data-testid="saveAssociatedTag"]').should('be.visible').click();
+    verifyResponseStatusCode('@saveTag', 400);
+    toastNotification(
+      `Tag labels ${glossary}.${term2} and ${glossary}.${term1} are mutually exclusive and can't be assigned together`
+    );
+
+    // Add non mutually exclusive tags
+    cy.get('[data-testid="tag-container"] [data-testid="tags"]')
+      .eq(0)
+      .should('be.visible')
+      .click();
```

Comment:
it will select 1st tag, there might be a chance it will become flaky, can you search for specific tags and select it?

Reply:
@ShaileshParmar11  this is for selecting the child element from the parent which will open the select dropdown. And for tags, we are doing the search and select pattern.